packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -937,31 +937,42 @@ const Call = SparkPlugin.extend({
   },
 
   /**
-   * The response to a PUT to LOCUS/media may not be fully up-to-dat when we
+   * The response to a PUT to LOCUS/media may not be fully up-to-date when we
```

Comment:
Did you want to put the prop in the jsdoc? No biggie if it is going to change soon